lib/Wrapper/OptSchedMachineWrapper.cpp

```diff
@@ -67,10 +76,12 @@ void OptSchedMachineModel::convertMachineModel(
 
   if (shouldGenerateMM) {
     if (mdlName_ == "ARM-Cortex-A7")
-      MMGen = createCortexA7MMGenerator(&dag, this);
+      MMGen = createCortexA7MMGenerator(&Dag, this);
+    else if (ExactCPU == "cortex-a53")
+      MMGen = createCortexA53MMGenerator(&Dag, this);
     else
-      Logger::Error("Could not find machine model generator for target \"%s\"",
-                    mdlName_.c_str());
+      llvm::report_fatal_error(
+          "Could not find machine model generator for target " + mdlName_);
```

Comment:
I think you can pass `false` to turn off the diagnostic message that states you can report the error to LLVM.
```
      llvm::report_fatal_error(
          "Could not find machine model generator for target " + mdlName_, false);
```